plugins/ufm_consumer_plugin/build/docker_build.sh

```diff
@@ -110,8 +110,10 @@ cp Dockerfile ${BUILD_DIR}
 cp -r ../../${PLUGIN_NAME}_plugin ${BUILD_DIR}
 # copy latest release of UFM to be used for installation
 #scp /mswg/release/ufm/ufm-enterprise/latest/ufm-*.ubuntu20.x86_64.mofed5.tgz ${BUILD_DIR}
-scp /mswg/release/ufm/ufm-enterprise/6.16.5/6.16.5-6/ufm-6.16.5-6.ubuntu20.x86_64.mofed5.tgz ${BUILD_DIR}
+#scp /mswg/release/ufm/ufm-enterprise/6.16.5/6.16.5-6/ufm-6.16.5-6.ubuntu20.x86_64.mofed5.tgz ${BUILD_DIR}
+#scp /auto/UFM/tmp/UFM-CI/13219/ubuntu20/ufm-6.16.5-4.ubuntu20.x86_64.mofed5.tgz ${BUILD_DIR}
 
+scp /mswg/release/ufm/ufm-enterprise/6.17.0/6.17.0-5/ufm-6.17.0-5.ubuntu20.x86_64.mofed5.tgz ${BUILD_DIR}
```

Comment:
need to update the plugin to use the latest RC build - UFM6.17.0-6 and create the final image on top of it.